Firestore/core/src/firebase/firestore/util/async_queue.cc

```diff
@@ -166,6 +166,10 @@ void AsyncQueue::RunScheduledOperationsUntil(const TimerId last_timer_id) {
   });
 }
 
+void AsyncQueue::SkipDelaysForTimerId(TimerId timer_id) {
```

Comment:
This doesn't seem to handle `SkipDelaysForTimeId(TimerId::All)`.

Reply:
I've added a comment to note that it does not.